images/spin-kitchensink/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=${BUILDPLATFORM} rust:1.59 AS build
```

Comment:
Considering this was a remanent from when we using `buildx`, we might want to remove it from the other Dockerfiles too.